storage/range_tree_test.go

```diff
@@ -46,3 +68,443 @@ func TestIsRed(t *testing.T) {
 		}
 	}
 }
+
+// TestFlip ensures that flips function correctly.
+func TestFlip(t *testing.T) {
```

Comment:
I'd rather leave it in for now.  At least until after I get the deletion and parent key work finished.